Dockerfile

```diff
@@ -10,106 +9,183 @@ COPY test /usr/src/app/test
 COPY src /usr/src/app/src
 RUN npm run build
 
+########################################################
+# Plugins
+########################################################
+
 FROM base AS plugin-builder
 
+# hadolint ignore=DL3008
 RUN apt-get update \
-	&& apt-get install \
+	&& apt-get install -y --no-install-recommends \
+	git \
 	libssl-dev \
 	openvpn \
 	&& rm -rf /var/lib/apt/lists/*
 
+########################################################
+# Connect-Disconnect Plugin
+########################################################
+
 FROM plugin-builder AS connect-disconnect-plugin
 
-ENV CONNECT_DISCONNECT_PLUGIN_COMMIT=7c958d8b33a87a06b5a8fa096397fc623494013a
-RUN git clone https://github.com/balena-io-modules/connect-disconnect-script-openvpn.git \
-	&& cd connect-disconnect-script-openvpn \
+ARG CONNECT_DISCONNECT_PLUGIN_COMMIT=7c958d8b33a87a06b5a8fa096397fc623494013a
+
+WORKDIR /usr/src/app/connect-disconnect-script-openvpn
+RUN git clone https://github.com/balena-io-modules/connect-disconnect-script-openvpn.git . \
 	&& git checkout ${CONNECT_DISCONNECT_PLUGIN_COMMIT} \
 	&& C_INCLUDE_PATH=/usr/include/openvpn/ make plugin
 
+########################################################
+# Learn-Address Plugin
+########################################################
+
 FROM plugin-builder AS learn-address-plugin
 
-ENV LEARN_ADDRESS_PLUGIN_COMMIT=8181b15c11dcbf437d1ea53eebf1dec75082f495
-RUN git clone https://github.com/balena-io-modules/learn-address-script-openvpn.git \
-	&& cd learn-address-script-openvpn \
+ARG LEARN_ADDRESS_PLUGIN_COMMIT=8181b15c11dcbf437d1ea53eebf1dec75082f495
+
+WORKDIR /usr/src/app/learn-address-script-openvpn
+RUN git clone https://github.com/balena-io-modules/learn-address-script-openvpn.git . \
 	&& git checkout ${LEARN_ADDRESS_PLUGIN_COMMIT} \
 	&& C_INCLUDE_PATH=/usr/include/openvpn/ make plugin
 
+########################################################
+# Auth Plugin
+########################################################
+
 FROM plugin-builder AS auth-plugin
 
-ENV AUTH_PLUGIN_COMMIT=623982a5d63dd2b7b2b9f9295d10d96a56d58894
-RUN git clone https://github.com/fac/auth-script-openvpn.git \
-	&& cd auth-script-openvpn \
-	&& git checkout ${AUTH_PLUGIN_COMMIT} \
+ARG OAS_PLUGIN_COMMIT=623982a5d63dd2b7b2b9f9295d10d96a56d58894
```

Comment:
Renamed to avoid Docker lint warnings to not use ARGS and ENVS for secrets (we had AUTH in the key).

OAS_ comes from "OpenVPN Auth Script"